website/docs/installing.mdx

```diff
@@ -18,7 +18,7 @@ npm i react-native-share --save
 
 After that, we need to install the dependencies to use the project on iOS(you can skip this part, if you are using this on Android).
 
-Now run a simple: `npx pod-install` or `cd ios && pod install`. After that, you should be able to use the library on both Platforms, iOS and Android. 
```

Comment:
Why are you adding these spaces?

Reply:
It is the auto-linting in webstorm - trailing spaces are usually deleted - https://eslint.org/docs/latest/rules/no-trailing-spaces